### PR TITLE
CHG Lectures.jsx to follow the format of other public pages. ADD LectureItem.styles.jsx and LectureItem.jsx. 

### PR DESCRIPTION
**Pull request by AaronBinChen**
Add LectureItem.jsx and LectureItem.styles.jsx with component LectureItem to follow format of other public pages.
BUG with file attachment. I tested with sample data and file attachment had issues showing up.

### CODE REVIEW

File: web/src/pages/core/public/Lectures/Lectures.jsx [hunk 1]
```diff
@@ -31,45 +35,62 @@ const Lectures = () => {
   }, []);
 
   const get_href = (row) => (
-    `${window.location.origin}/api/public/static${row.static_file.path}/${row.static_file.filename}`
+    `/api/public/static${row.static_file.path}/${row.static_file.filename}`
   );
-
+  console.log(lectures);
```
Comment (synoet) [inline web/src/pages/core/public/Lectures/Lectures.jsx:40]: remove the console.log

File: web/src/pages/core/public/Lectures/Lectures.jsx [hunk 2]
```diff
@@ -31,45 +35,62 @@ const Lectures = () => {
...
+//               rows={lectures}
+//               rowsPerPageOptions={[10, 20, 30]}
+//               pagination
+//             />
+//           </Paper>
+//         </Grid>
+//       </Grid>
+//     </StandardLayout>
+//   );
+// };
```
Comment (synoet) [inline web/src/pages/core/public/Lectures/Lectures.jsx:94]: Please remove the comments, from the pr that arent needed. I you are ever curious about a previous state you can look at the history.

**Comment by synoet** [comment]
Please fix the small comments. Then you will be good to merge :100:

**Comment by AaronBinChen** [comment]
The latest commit to address Issue #352 is 9d4512b. Anything beyond that commit is related to another issue.

**Comment by wabscale** [comment]
Rebased and squashed https://github.com/AnubisLMS/Anubis/commit/46b5f14ea42e9a5fd0f3778117a961c2f8e6e78c

**Pull request closed without merging**